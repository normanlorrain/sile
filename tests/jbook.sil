\begin[class=jbook]{document}
\end{document}
