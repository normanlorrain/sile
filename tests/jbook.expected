Set paper size 	595.275597	841.8897729
Begin page
Mx 	295.2916
My 	780.6177
Set font 	Gentium Plus;10;400;;normal;;LTR
T	20	(1)
End page
Finish
